Graphics/GraphicsEngineOpenGL/src/TextureBaseGL.cpp

```diff
@@ -493,6 +493,28 @@ void TextureBaseGL::CreateViewInternal(const TextureViewDesc& OrigViewDesc, ITex
                 DEV_CHECK_GL_ERROR_AND_THROW("Failed to create texture view");
                 pViewOGL->SetBindTarget(GLViewTarget);
 
+                if (ViewDesc.Format == TEX_FORMAT_X24_TYPELESS_G8_UINT || ViewDesc.Format == TEX_FORMAT_X32_TYPELESS_G8X24_UINT)
+                {
+                    const auto &FormatInfo = pDeviceGLImpl->GetTextureFormatInfo(ViewDesc.Format);
```

Comment:
We are trying to avoid excessive use of `auto` now for better code readability.
Let use the actual type here.